inference-engine/thirdparty/clDNN/api/engine.hpp

```diff
@@ -61,6 +62,7 @@ struct engine_configuration {
                                               ///< (switched off for older drivers then NEO).
     uint16_t n_streams;                       ///< Number of queues executed in parallel
     const std::string kernels_cache_path;     ///< Path to compiled kernels cache
+    uint16_t n_threads; ///< Number of threads used to build program
```

Comment:
I suppose it makes sense to align this comment as others.

Reply:
Applied